### DISCUSSION TITLE
Last update custom FBW - huge issue - mod not working

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4247] [General]
Hi simmers,

after the last update, the custom fbw mod is not working. I have the original Asobo panel instead of the mod??? Anyone else with this problem?

 ...developer mod is also not working.... trying the stable version ... the stable is producing a CTD :-( .... as described in another thread ...

I changed the size of the pagefile. But this could not be the problem ?! hmmm ...

Greets Klaus

**Comment by roxter69** [comment]
same hear cptbayer

**Reply by CptBayer** [reply]
ah ok.... I think I know why.... in the aircraft selection menu is a new option, better a separate field for the flybywire airbus. If you select this, no aircraft model no panel. If you select the other airbus, you will get the Asobos panel ...

![grafik](https://user-images.githubusercontent.com/76531893/113482974-59abaa00-94a1-11eb-9f51-33de6706fa80.png)


greets Klaus

**Reply by roxter69** [reply]
cptbayer thnx so much im sorted,im sure there will be a few more that will find your post useful thnx again happy flying

**Comment by PatriceCesson** [comment]
So, is it ok or not?

**Reply by CptBayer** [reply]
not yet fixed ...

greets Klaus

**Comment by ExampleWasTaken** [comment]
There is an update for the installer. Download that, then delete the old A32NX folder from your community folder, and reinstall the version you want.

**Reply by CptBayer** [reply]
yeah thanx. Working again. Now the master question: How to implement other liveries? 

Greets Klaus

**Reply by ExampleWasTaken** [reply]
If you want to convert your own liveries [this](https://docs.flybywiresim.com/start/convert-liveries/) link might help you. Be aware that liveries might be subject to copyright and may not be modified by any other than the author.

**Reply by CptBayer** [reply]
Hi,


thanx for the info. So this means at this time I can´t add textures for example downloaded from flightsim.to?! In P3D I added simply the needed information in the aircraft.cfg. YOu know what I mean [fltsim.xy] ...

Forget my thread... I´ll try your link .... 

Greets Klaus

**Reply by ExampleWasTaken** [reply]
You can edit the necessary files. Just know that some liveries may not be modified. It all depends on what the author specified.

**Reply by AdamNZ** [reply]
I've made the odd A320 livery. Where can I find info about what to modify? TIA.

EDIT: Here we go!
https://docs.flybywiresim.com/start/convert-liveries/

**Reply by CptBayer** [reply]
it is workiong great..... Can anybody code a bat file :-) ?! 

Greets Klaus

**Reply by ContactRon5** [reply]
Hi. I've been having a play around all morning trying to get this to work. I ended up deleting everything, including installer, and starting afresh. There appears to be several issues.
1. This doesn't work for the stable installation. You still only have 3 airliners (A320, 747, 787) in MSFS and the FlyByWire version has the default msfs panel.
2. You only see the 4 airliners if you install the development and above versions.
3. The system seems to work when you only have installed a development aircraft with default FlyByWire livery. I had a BA livery and followed the instructions on how to assign the FBW panel. The aircraft appeared on the liveries which could be selected, but the yellow 'FLY' in msfs tab was blanked out. 
4. The instructions on how to adapt 3rd party liveries were a little confusing. STEP 3  asks you to edit the model.cfg file.  The problem is there are two model.cfg files - one is in directory MODEL.MAIN the other being MODEL.<livery> i.e. MODEL.AIRFRANCE.   I could edit the two lines of the .cfg file in MODEL.MAIN as instructed, but as I say, when you went to fly the aircraft, the yellow FLY tab was blank.
5. If I then edited the model.cfg file in, for example, MODEL.AIRFRANCE, which is not part of the instructions, the yellow FLY tab would be available and I was able to continue.
6. When I say continue...   that was very, very unreliable and often resulted in the whole simulator crashing.

So in summary, the only aircraft / livery I can fly at present is the default development FlyByWire livery.

( Hope the above isn't too confusing! )

**Reply by ExampleWasTaken** [reply]
@ContactRon5 
Point 1 and 2 are expected and have been outlined on the Discord and Twitter. 
Please note that the stable rightfully has the default cockpit textures.
Regarding the livery conversion, that is up to each livery creator. The documentation is only a rough guide.
